packages/x-charts/src/hooks/useChartRootRef.ts

```diff
@@ -0,0 +1,13 @@
+'use client';
+import * as React from 'react';
+import { useChartContext } from '../context/ChartProvider';
+
+/**
+ * Get the ref for the root chart element.
+ * @returns The root chart element ref.
+ */
+export function useChartRootRef(): React.RefObject<HTMLDivElement | null> {
```

Comment:
Followed the logic of `svgRef`. We need to access the chart root when printing otherwise the legend would be left out of the print.